app/resources/add_note_base.html.template

```diff
@@ -12,6 +12,21 @@
 
 {% load i18n %}
 
+{% if env.enable_javascript %}
+   <script type="text/javascript">
+   jQuery(document).ready(function(){
+     jQuery("div.status-expand").hide();
+     jQuery("a.status-button").click(function(){
+       if (jQuery("div.status-expand").is(":hidden"))
```

Comment:
Replace :hidden with :visible here and swap the statements below. It's usually easier to read by putting a positive condition before a negative one.